scormplayer/scorm-tool/src/java/org/sakaiproject/scorm/ui/console/components/ContentPackageDetailPanel.java

```diff
@@ -37,8 +38,9 @@ public ContentPackageDetailPanel(String id, ContentPackage contentPackage)
 	{
 		super(id, new TypeAwareCompoundPropertyModel(contentPackage));
 
-		String createdByName = getString("unknown");
-		String modifiedByName = createdByName;
+		String unknownUser = new ResourceModel("unknown", "unknown").getObject();
```

Comment:
seems like this should be fixed

Reply:
it's fixing a wicket-ism of where a Model is instantiated .... init() is the wrong place and you get wicket errors in logs.